.github/workflows/velox_be.yml

```diff
@@ -38,91 +38,94 @@ on:
       - 'substrait/substrait-spark/**'
 
 env:
-  HTTP_PROXY_HOST: 10.240.252.16
+  HTTP_PROXY_HOST: proxy-shz.intel.com
   HTTP_PROXY_PORT: 911
-  DOCKER_REGISTRY_HOST: 10.1.0.25
-  DOCKER_REGISTRY_PORT: 5000
+  PATH_TO_GLUTEN_TE: ./tools/gluten-te
+  DOCKER_PUSH_REGISTRY:
+  DOCKER_CACHE_REGISTRY: 10.1.0.25:5000
+  LEGACY_DOCKER_REGISTRY: 10.0.2.4:5000
 
 concurrency:
   group: ${{ github.repository }}-${{ github.head_ref || github.sha }}-${{ github.workflow }}
   cancel-in-progress: true
 
 jobs:
-
   ubuntu2004-test-spark32:
     runs-on: velox-self-hosted
+    env:
+      OS_IMAGE_NAME: ubuntu
+      OS_IMAGE_TAG: 20.04
     steps:
       - uses: actions/checkout@v4
       - name: Setup docker container
         run: |
-          docker run --rm --init --privileged --ulimit nofile=65536:65536 --ulimit core=-1 --security-opt seccomp=unconfined \
-          -v $PWD:/opt/gluten --name ubuntu2004-test-$GITHUB_RUN_ID -e NUM_THREADS=30 -detach $DOCKER_REGISTRY_HOST:$DOCKER_REGISTRY_PORT/gluten-te/gluten-build-ubuntu:20.04 \
-          'cd /opt/gluten && sleep 14400'
+          $PATH_TO_GLUTEN_TE/$OS_IMAGE_NAME/gha/gha-checkout/checkout.sh
       - name: Build Gluten velox third party
         run: |
-          docker exec ubuntu2004-test-$GITHUB_RUN_ID bash -c '
+          $PATH_TO_GLUTEN_TE/$OS_IMAGE_NAME/gha/gha-checkout/exec.sh '
           cd /opt/gluten/ep/build-velox/src && \
           ./get_velox.sh --velox_home=/opt/velox && \
           ./build_velox.sh --velox_home=/opt/velox --enable_ep_cache=ON --build_test_utils=ON'
```

Comment:
We should be able to set `run_setup_scipts=OFF` as all dependencies are already installed